jalib/jalib.cpp

```diff
@@ -34,29 +34,56 @@
 #include "jalib.h"
 #include "jassert.h"
 
-jalib::JalibFuncPtrs jalib::jalibFuncPtrs;
-int jalib::jalib_funcptrs_initialized = 0;
-const char *jalib::elfInterpreter = NULL;
-int jalib::stderrFd = -1;
-int jalib::logFd = -1;
-int jalib::dmtcp_fail_rc = -1;
+static jalib::JalibFuncPtrs jalibFuncPtrs;
+static int jalib_funcptrs_initialized = 0;
+
+static struct {
+  const char *elfInterpreter;
+  int stderrFd;
+  int logFd;
+  int dmtcp_fail_rc;
+} dmtcpInfo = {NULL, -1, -1, -1};
```

Comment:
Because dmtcpInfo occurs before the call to `namespace jalib`, it is part of the std namespace.  This is also the case for `jalib_funcptrs_initialized`.  However, `jalibFuncPtrs` is explicitly made a member of the `jalib` class.  All three are static, and so we can do anything we like, since they won't be exposed outside of this file.
However, I would prefer a common convention.  Either we should make everything part of `jalib` or none of this.  (Naturally, it's fine to break this convention when we have a good reason, but we should start with a standard convention.)